### ISSUE TITLE
[Accessibility][Screen reader-Button]:There is no label associated with header element for 'Standard and Primary buttons' under default button section

### ISSUE THREAD

**Issue by dollykumari5** [OfficeDev/office-ui-fabric-react#9687]
Detailed description of this issue can be found here [SharePoint](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/ES4u1n2IVNNPuoO2dU5gywIBdNy61RcmtbihpLEKSTfzyA?e=M7f2Dv)

Component:Button

**Comment by khmakoto** [comment]
This issue seems like it's going to be fixed by #9697, so I'll close this as a dupe of #9685.
